packages/wallet/.eslintrc.js

```diff
@@ -1,49 +1,89 @@
+const generalRules = {
+  "no-var": "off",
+  "no-async-promise-executor": "off",
+  "no-case-declarations": "off",
+  "no-extra-boolean-cast": "off",
+  "no-undef": "off" // Shouldn't need this, tsc takes care of it
+};
+
+// From the tslint.json we used previously
+const leftoverTsLintRules = {
+  "@typescript-eslint/no-explicit-any": "off",
+  "@typescript-eslint/explicit-function-return-type": "off",
+  "@typescript-eslint/no-use-before-define": "off",
+  "@typescript-eslint/ban-ts-ignore": "off",
+  "@typescript-eslint/no-non-null-assertion": "off",
+  // TODO: Get rid of this, there are just a small number of cases
+  "@typescript-eslint/no-unused-vars": "off"
+};
```

Comment:
I think we should reexamine these in the future. Especially `no-explicit-any` and `explicit-function-return-type` but pretty much all of them...

We are losing out on the things that Typescript offers us when we do these things.

Reply:
I agree. This PR is not meant to change any rules, it's just trying to recreate what we had before.